src/FirstSteps/RPG/Heroes/Boss.cs

```diff
@@ -0,0 +1,36 @@
+﻿
+using System.Dynamic;
+
+namespace FirstSteps.RPG.Heroes
+{
+    public class Boss : IAttacker
+    {
+        public string Name { get { return _name; } }
```

Comment:
It should look like this:
```csharp
private string _name;
public int Health { get; private set; }
public int Strength { get; private set; }
public int Damage { get; private set; }
```
Always try to set fields with minimum external access needed. You don't need a field externally: use a private variable, you need to display a value or something: use public property with the public get only.